src/__snapshots__/Scanner.test.js.snap

```diff
@@ -212,7 +212,6 @@ exports[`performance should stop scanning if page grows above maximum forms 1`]
       for="input-01"
     >
       <input
-        data-ddg-inputtype="unknown"
```

Comment:
The new `stopScanner` method actively cleans up all this stuff, so the snapshot needed to be updated.